test/e2e/performanceprofile/functests/utils/nodes/nodes.go

```diff
@@ -275,7 +276,7 @@ func GetSMTLevel(cpuID int, node *corev1.Node) int {
 
 // GetNumaNodes returns the number of numa nodes and the associated cpus as list on the node
 func GetNumaNodes(node *corev1.Node) (map[int][]int, error) {
-	lscpuCmd := []string{"lscpu", "-e=cpu,node", "-J"}
+	lscpuCmd := []string{"lscpu", "-e=node,core,cpu", "-J"}
```

Comment:
not needed.

Reply:
We need it because, we want lscpu to give us the extended output in the order asked and i  use this order using the -J(json) and then dump(unmarshal) it in the structures fields.

Reply:
understood.